packages/block-editor/src/components/inserter/no-results.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * WordPress dependencies
+ */
+import { createInterpolateElement } from '@wordpress/element';
+import { __, sprintf } from '@wordpress/i18n';
+import { Icon, blockDefault } from '@wordpress/icons';
+
+function InserterNoResults( { filterValue } ) {
+	return (
+		<div className="block-editor-inserter__no-results">
+			<Icon
+				className="block-editor-inserter__no-results-icon"
+				icon={ blockDefault }
+			/>
+			<p>
+				{ createInterpolateElement(
+					sprintf(
+						/* translators: %s: search term. */
+						__( 'Sorry no results found on <strong>%s</strong>.' ),
```

Comment:
It might do for a copy review. At least, from my perspective, I'd expect a comma after "Sorry", or to omit "Sorry" altogether, and that "on" is a bit ambiguous in lacking some context such as "on the search term ...".

```suggestion
						__( 'No results found for search term: <strong>%s</strong>.' ),
```

Or, do we care to include the term, if it's already visible in the search field?

```suggestion
						__( 'No results found.' ),
```

Reply:
I say omit the search term. I would assume the user already knows what they typed into the search bar.

Reply:
Context: that this is based on the design proposal by @pablohoneyhoney

Reply:
committed, we can  update later if needed.